pkg/trace/api/info.go

```diff
@@ -64,19 +63,19 @@ func (r *HTTPReceiver) makeInfoHandler() (hash string, handler http.HandlerFunc)
 		oconf.Memcached = o.Memcached.Enabled
 	}
 	txt, err := json.MarshalIndent(struct {
-		Version          string        `json:"version"`
-		GitCommit        string        `json:"git_commit"`
-		BuildDate        string        `json:"build_date"`
+		Version   string `json:"version"`
+		GitCommit string `json:"git_commit"`
+		// BuildDate        string        `json:"build_date"`
 		Endpoints        []string      `json:"endpoints"`
 		FeatureFlags     []string      `json:"feature_flags,omitempty"`
 		ClientDropP0s    bool          `json:"client_drop_p0s"`
 		SpanMetaStructs  bool          `json:"span_meta_structs"`
 		LongRunningSpans bool          `json:"long_running_spans"`
 		Config           reducedConfig `json:"config"`
 	}{
-		Version:          info.Version,
-		GitCommit:        info.GitCommit,
-		BuildDate:        info.BuildDate,
+		Version:   r.conf.AgentVersion,
+		GitCommit: r.conf.GitCommit,
+		// BuildDate:        info.BuildDate, // TODO: this would be a breaking change, it doesn't appear this is used by any of the tracers I searched through
```

Comment:
Correct, let's remove it. We shouldn't have commented out code. Only the `hash` is used to identify changes in this config.